accelerator/apis.c

```diff
@@ -132,9 +133,10 @@ status_t api_generate_address(const iota_config_t* const iconf, const iota_clien
   }
 
   lock_handle_lock(&cjson_lock);
-  ret = ta_generate_address(iconf, service, res);
+  ret = ta_generate_address_helper(iconf, service, res);
```

Comment:
I have no clue why you would change it with helper function here.

Reply:
I've misunderstood the role between helper function and the original function. The helper function should perform the original functionality, and `ta_generate_address` wraps the helper function.